### PR TITLE
Tdr 257 sbt release job

### PR DESCRIPTION
**Pull request by TomJKing**

### CODE REVIEW

File: vars/sbtReleaseDeployJob.groovy [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  pipeline {
+    agent {
+      label "master"
+    }
+    //Parameters section required for display in client Jenkins jobs GUI.
+    //Selected values are not accessible within the function and must be passed into the function via the config map
+    parameters {
+      choice(name: "STAGE", choices: ["intg", "staging", "prod"], description: "The stage you are deploying the ${config.libraryName} library to")
+    }
+    stages {
```
Comment (MancunianSam) [inline vars/sbtReleaseDeployJob.groovy:15]: Do you need this parameters block if you're passing in the stage as the parameter map?

**Comment by MancunianSam** [comment]
Looks good mate, just one question

File: vars/sbtReleaseDeployJob.groovy (hunk 1 above)
Comment (TomJKing) [reply vars/sbtReleaseDeployJob.groovy:15]: Its for the client Jenkins job, to display the parameters if a manual build is required in the Jenkins GUI. For example deploying to staging/prod. See for example: https://jenkins.tdr-management.nationalarchives.gov.uk/job/TDR%20Auth%20Utils%20Deploy/

Comment (MancunianSam) [reply vars/sbtReleaseDeployJob.groovy:15]: Ok cool, makes sense.

**Review by MancunianSam — APPROVED** [review]

**Pull request merged**